tests/integration/bash_hook.rs

```diff
@@ -2,25 +2,5 @@ use crate::integration::register_test;
 
 #[test]
 fn command_bash_hook_help() {
```

Comment:
This function should probably be renamed.

Reply:
indeed